_posts/2020-08-21-running-tutorials.md

```diff
@@ -0,0 +1,340 @@
+---
+layout: post
+title: Running tutorials
+tagline: How we run community tutorials
+author: Jacob Tomlinson (NVIDIA)
+tags: [Tutorials, Community]
+theme: twitter
+---
+
+{% include JB/setup %}
+
+For the last couple of months we've been running community tutorials every three weeks or so. The response from the community has been great and we've had 50-100 people at each 90 minute session.
+
+## Why should open source projects run tutorials
+
+The Dask team has historically run tutorials at conferences such as SciPy. With 2020 turning out the way that it has much of this content is being presented virtually this year. As more people are becoming accustom to participating in virtual tutorials we felt it would be a good service to our community to start running regular virtual tutorials independent of conferences we may be attending or speaking at.
```

Comment:
```suggestion
The Dask team has historically run tutorials at conferences such as SciPy. With 2020 turning out the way that it has much of this content is being presented virtually this year. As more people are becoming accustomed to participating in virtual tutorials we felt it would be a good service to our community to start running regular virtual tutorials independent of conferences we may be attending or speaking at.
```